prison-core/src/main/java/tech/mcprison/prison/internal/Player.java

```diff
@@ -77,7 +78,9 @@
      */
     void setScoreboard(Scoreboard scoreboard);
 
+    Gamemode getGamemode();
 
+    void setGamemode(Gamemode gamemode);
```

Comment:
While I do agree with this and will be committing to implement, half of prison-core doesn't have documentation. I'll be happy to write the missing documentation, I just find it a bit hypocritical.